analysis_driver/pipelines/demultiplexing.py

```diff
@@ -49,8 +50,13 @@ def demultiplexing_pipeline(dataset):
     crawler.send_data()
     dataset.end_stage('setup')
 
-    # Need to sleep to make sure the LIMS has had enough time to update itself
-    time.sleep(900)
```

Comment:
After Chatting with Murray we agreed that we should bring down the sleep time to 20min (approx 1 cycle)